src/components/TablePackages/TablePackageHeaders.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, TableCell, Typography } from "@mui/material";
+import { PackageHeadersValues } from "./types";
+
+const PackageTableHeaders = ({ headers }: PackageHeadersValues) => {
+  return (
+    <>
+      {headers.map((element) => {
+        return (
+          <>
+            <TableCell align="center">
+              <Box borderRight={"1px solid gray"}>
```

Comment:
I think it should be possible to add this border as `sx` prop to TableCell, then this Box wont be needed anymore

Reply:
It doesn't produce this effect. I changed padding of TableCell and now it should be good.